### DISCUSSION TITLE
v2.4.0 Beta - New Design + Unified Library + New features and improvements

### DISCUSSION THREAD

**Discussion by flavioislima** [Heroic-Games-Launcher/HeroicGamesLauncher#1478] [Q&A]
Hello! 
This is the first beta for the next stable release and already brings several new features and improvements.
In summary, we have design changes, the current download/update will be shown on the sidebar all the time, even if you leave the library and go to the store or settings, etc; the game library was overhauled and now the game cards will show the game title all the time as well the buttons for play, settings, and update (if any). 
This is probably a change that some people won't like but this is good for accessibility and we know some games don't show the name on the cover. But let's wait for the feedback.
Besides that, now we have a unified library with all games from GOG and Epic if you have logged in to both. 
A favorites button to show only the favorites, an auto-complete feature for the search, and more!

### Changed
* [UI/UX] Library improvements by @flavioislima in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1449
  - Add the current Download/Update(s) on the sidebar;
  - Show a unified library + add a new Store Filter. 
  - Add the ability to collapse the sidebar.
  - Updated the Sidebar with a new design;
  - Add a Favorites button to show only the favorites;
  - Add Auto-Complete to the Search Bar;
  - Simplify Game List for a cleaner look;
  - Use this new game list for the wine manager as well;
  - Add scale effect on GameCard on mouse hover. 
  - Change the Design of buttons, selects, and inputs to match the current Figma Design.
  - Fine-tune the fuzzy search since it was showing a lot of items not related to the search.
  - Several styles fixes to improve consistency;
  - Add the Heroic version to the sidebar;
  - Added a notification when starting moving a game
* [UX] Add option to change Game Language by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1440
* [Fix] Disable controller on boot setting by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1438
* [Fix] detection of search bar focus to open virtual keyboard by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1450
* [Fix] DOSBOX Windows Games by @imLinguin and @flavioislima in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1469
* [Fix] Setup GOG games with Proton + ScummVM patch by @imLinguin and @flavioislima in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1472
* Fix] multiple settings writes by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1464
* [Linux] Added option to display Wine executable path by @Delta2Force in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1409
* [Fix] No longer replace token/sid with redacted by @CommandMC in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1442
* [Other] Remove CrossOver support on Linux by @CommandMC in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1404
* [Tech] Pass SDL list as multiple arguments by @CommandMC in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1443
* [Tech] Store only game setting keys in game config file during installation by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1467
* [Tech] Restructure/Cleanup of runLegendaryOrGogdlCommand by @Nocccer in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1415
* Translations update from Hosted Weblate by @weblate in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1466

## New Contributors
* @Delta2Force made their first contribution in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1409

### Screenshots

![image](https://user-images.githubusercontent.com/26871415/172703432-55067692-6c53-4a85-bdb5-dd47aae3b0ca.png)
![image](https://user-images.githubusercontent.com/26871415/172704266-f37fe194-73d6-44e1-a33b-ee6d7b61117b.png)
![image](https://user-images.githubusercontent.com/26871415/172832066-76f05531-a5e3-4940-a65a-3e2fb69ee431.png)
![image](https://user-images.githubusercontent.com/26871415/172703625-88bfab6e-155a-4a01-a4c5-7b3ba8cdf3fb.png)

**Full Changelog**: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/compare/v2.3.8...v2.4.0-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/tag/v2.4.0-beta'>v2.4.0 Beta - New Design + Unified Library + New features and improvements</a>.</em>

**Comment by TheOnlyBrad** [comment]
Hi there.. Adding the new feature "Change game language" was a very fine idea, but i'm having the following problems with it on Steam Deck (tested with the game "Prey" from Epic Store"): The feature only works fine, as long as i launch the game directly via Heroic Games Launcher in Desktop Mode. Importing it to Steam (via "Add non Steam game") the game remains starting in Steam standard language (=English). I would like it to use German speach and GUI, but this unfortunately doesn't work inside Steam (in Gaming mode). Any thoughts on this..? Is there maybe a special start prefix i have to use inside Steam Gaming mode?

**Reply by Nocccer** [reply]
I don't know why this happens but legendary can't pick up the locale or language setting if you run it inside steam. Didn't find a workaround yet.
